Project2015To2017/Reading/ProjectReader.cs

```diff
@@ -15,6 +15,7 @@ public sealed class ProjectReader
 		private readonly Caching.IProjectCache projectCache;
 		private readonly NuSpecReader nuspecReader;
 		private readonly AssemblyInfoReader assemblyInfoReader;
+		private readonly ProjectPropertiesReader _projectPropertiesReader;
```

Comment:
I intend later to rename other private instance fields to start with `_`, to match common C# codestyle conventions.

Reply:
That depends on who you ask, I never prefix with `_`. I think we should add a .editorconfig instead for consistency. It doesn't matter what you do as long as it's consistent.

Reply:
I'll make it back consistent in the merge let's spin up an issue of adding an editorconfig and getting everything in line.